node_lambnik/scripts/fetch-data.sh

```diff
@@ -0,0 +1,14 @@
+#!/usr/bin/env bash
+
+# Fetches development data sets from OpenDataPhilly
+if ! [ -f data/STR_Centerline.zip ]; then
```

Comment:
The filename matches the table name now, but this check is still looking for the old filename (so it re-downloads every time).